sankeys/views.py

```
@@ -0,0 +1,83 @@
+        #FinancialFlow.objects.filter(year=year).delete()
+
+        service = FinancialFlowService()
+        companies = Company.objects.all()
+        success_count = 0
+
+        for company in companies:
+            try:
+                # 보정 로직이 포함된 서비스 호출
+                if service.fetch_and_save_financials(company, year):
+                    success_count += 1
+            except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)